earthly

```diff
@@ -127,7 +127,7 @@ case "$1" in
         if [ -z "$COMP_LINE" ]; then
             update="false"
             if [ "$last_flag_hash" != "$flagoverride_hash" ]; then
-                echo ".earthly_version_flag_overrides has changed since last run, checking for prerelease binaries" \
+                echo ".earthly_version_flag_overrides has changed since last run, checking for prerelease binaries." \
```

Comment:
```suggestion
                echo ".earthly_version_flag_overrides has changed since last run, checking for prerelease binaries. " \
```

Reply:
I wondered about that, but for some reason it didn't seem to be necessary. That said, it doesn't bother me.